### PR TITLE
PRS 13560 61 62 63

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Persons related to MY 002

### CODE REVIEW

File: new/PRS13561TasabkaMadhen.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተሰብከ፡ መድኅን፡</persName>
+                     <persName xml:lang="gez" type="normalized" corresp="#n1">Tasabka Madḫǝn</persName>
+                      <persName type="alt"><roleName type="title">ʿaqqabe saʿat</roleName> Tasabka Madḫǝn</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit notBefore="1370" notAfter="1430"/>
```
Comment (eu-genia) [inline new/PRS13561TasabkaMadhen.xml:59]: maybe here also add the place of activity, or under floruit add as text 
```
He was  the abbot (NB: <foreign xml:lang="gez">ʿaqqabe saʿat</foreign>) of
                  <placeName ref="INS0039MY">Dabra Māʿṣo Qǝddus Yoḥannǝs</placeName> in the late 14th-early 15th century
```

File: new/PRS13562Aron.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">አሮን፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾAron</persName>
+               <persName type="alt"><roleName type="title">makʷannǝn</roleName> ʾAron</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ruler"/>
```
Comment (eu-genia) [inline new/PRS13562Aron.xml:52]: if it is known ruler of which area this would be helpful

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13561TasabkaMadhen.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS13561TasabkaMadhen.xml:59]: ok.

File: new/PRS13562Aron.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/PRS13562Aron.xml:52]: Done.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**